staging/src/k8s.io/apiserver/pkg/registry/generic/rest/streamer_test.go

```diff
@@ -28,6 +28,8 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
not going to ask for another change at this point but technically 3rd party deps get their own section...